builds/jenkins-delete-resources/README.md

```diff
@@ -0,0 +1,5 @@
+## Delete Resources for Multi Account and Multi region!
```

Comment:
I'm having trouble understanding where this is being used :) How do you trigger this JenkinsFile?

Reply:
This Jenkins file is triggered, while destroying the previous stack.This job get triggered from a python script where it gets the Jenkins urls and credentials as the input.

The Python script get executed just before the stack destroy stage.

Reply:
updated the Readme